govcd/tar.go

```diff
@@ -0,0 +1,106 @@
+package govcd
+
+import (
+	"archive/tar"
+	"errors"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+func Unpack(tarFile string) ([]string, error) {
+
+	var filePaths []string
+
+	reader, err := os.Open(tarFile)
+	if err != nil {
+		return filePaths, err
+	}
+	defer reader.Close()
+
+	tarReader := tar.NewReader(reader)
```

Comment:
Does this code block out-of-tree references? That's a vuln we had to fix in pyvcloud.  See https://github.com/vmware/pyvcloud/pull/268.

Reply:
Yes, we have similar additional functionality:
func sanitizedName(filename string) string {
	if len(filename) > 1 && filename[1] == ':' {
		filename = filename[2:]
	}
	filename = strings.Replace(filename, "\\/.", "\\/", -1)
	filename = strings.Replace(filename, "../", "", -1)
	return strings.Replace(filename, "..\\", "", -1)
}
Though I will add verify task of this security issue